apps/ae_cuckoo_cycle_pow_executables/src/ae_cuckoo_cycle_pow_executables.app.src

```diff
@@ -0,0 +1,15 @@
+{application, ae_cuckoo_cycle_pow_executables,
```

Comment:
this is way too long name. Also please follow the convention without underscore aecuckoo is fine.